<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObanalStatus</class>
 <widget class="QDialog" name="ObanalStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="aeroStat">
     <property name="text">
      <string>Выполняется...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="surfLbl">
     <property name="text">
      <string>Анализ приземных данных</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="aeroLbl">
     <property name="text">
      <string>Анализ аэрологических данных</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="gribStat">
     <property name="text">
      <string>Выполняется...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="gribLbl">
     <property name="text">
      <string>Анализ GRIB</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="surfStat">
     <property name="text">
      <string>Выполняется...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="closeBtn">
     <property name="toolTip">
      <string>Выход</string>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="oceanLbl">
     <property name="text">
      <string>Анализ гидрологических данных</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="oceanStat">
     <property name="text">
      <string>Выполняется...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="closeCheck">
     <property name="text">
      <string>Закрыть после завершения </string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
